api/migrations/20210813200559-create-bean-type.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("beanTypes", {
```

Comment:
Can we model bean types as an enum https://www.postgresql.org/docs/12/datatype-enum.html?

Reply:
From my knowledge, I think that beanTypes are better because in the future they will also let us access some beanSpecificData

Reply:
I see. Do bean specific data need to be in database, or can they be stored in code?